molecule/provisioner/ansible/playbooks/lxd/prepare.yml

```diff
@@ -51,3 +51,11 @@
           - curl
           - sudo
         state: present
+
+    - name: Install ca-certificates-cacert and ca-certificates-mozilla for openSUSE
+      package:
+        name:
+          - ca-certificates-cacert
+          - ca-certificates-mozilla
+        state: present
+      when: ansible_os_family == 'Suse'
```

Comment:
Do we need to also specify `ansible_distribution_version` here? 

I assume that these `ca-certificate-*` packages would not be available on older versions?

Reply:
Not really needed, because we just have images:opensuse/42.3 and images:opensuse/15.0 from LXD repo :-P

Reply:
Right, but if someone is building other older images, they will get a failed task, no? We need to be careful here and try to cover all the cases we can think of for this core functionality. Molecule use case are big and wide (just like Ansible!)!

Reply:
OK, looks resolved by https://github.com/ansible/molecule/pull/1740#issuecomment-463037332.